configs/llm-360-amber1.yaml

```
@@ -0,0 +1,602 @@
+run_name: amberish1-base
+seed: 6198
+dry_run: false
+
+wandb:
+  name: ${run_name}
+  project: olmo-small
+  group: amberish1-base
+
+model:
+  d_model: 2048
+  n_heads: 16
+  n_layers: 16
+  mlp_ratio: 8
+  # mlp_hidden_size: 22016
+  weight_tying: false
+  alibi: false
+  rope: true
+  flash_attention: true
+  attention_dropout: 0.0
+  attention_layer_norm: false
+  clip_qkv: null
+  include_bias: false
+  block_type: sequential
+  layer_norm_type: rms
+  layer_norm_with_affine: false
+  bias_for_layer_norm: false
+  attention_layer_norm_with_affine: false
+  activation_type: swiglu
+  residual_dropout: 0.0
+  embedding_dropout: 0.0
+  max_sequence_length: 2048
+  vocab_size: 32000
+  embedding_size: 32000
+  eos_token_id: 2
+  pad_token_id: 2
+  init_device: meta
+  init_fn: normal
+  init_std: 0.02
+  init_cutoff_factor: 3
+
+compile: null
+
+fused_loss: false
+
+optimizer:
+  name: adamw
+  learning_rate: 4.0e-4
+  weight_decay: 0.1
+  decay_norm_and_bias: true
+  decay_embeddings: false
+  epsilon: 1.0E-08
+  betas:
+  - 0.9
+  - 0.95
+  metrics_log_interval: 10
+
+scheduler:
+  name: cosine_with_warmup
+  units: tokens
+  t_warmup: 4587520000
+  t_max: 1.25e12
+  alpha_f: 0.1
+  warmup_min_lr: 0
+  grad_clip_warmup_steps: null
+  grad_clip_warmup_factor: 5
+  warmup_min_lr: 0
```

`warmup_min_lr` is included twice, and you could just exclude `grad_clip_warmup_*` for simplicity:
```suggestion
  warmup_min_lr: 0
```